application/src/ext/java/org/opentripplanner/ext/stopconsolidation/DecorateConsolidatedStopNames.java

```diff
@@ -51,6 +55,43 @@ private void replaceConsolidatedStops(Itinerary i) {
     });
   }
 
+  /**
+   * Removes walk legs from and to a consolidated stop if they are deemed "short". This means that
+   * they are from a different element of the consolidated stop.
+   */
+  private void removeWalkLegs(Itinerary itinerary) {
```

Comment:
Rename to `removeShortWalkLegs`, and maybe there is a way to consolidate the various conditions?